### DISCUSSION TITLE
Menu with hamburger icon

### DISCUSSION THREAD

**Discussion by ghost** [fomantic/Fomantic-UI#1380] [Collections (form, grid, menu, message, table, ...)] [answered]
Is it possible to have a kind of responsive menu for mobile and tablets so that a classic hamburger icon including dropdown is displayed?

**Comment by lubber-de** [comment, marked as answer]
This is only possible at the moment by using 2 different menus or using a `stackable` menu 

https://fomantic-ui.com/collections/menu.html#stackable
or using a sidebar
https://fomantic-ui.com/modules/sidebar.html
https://fomantic-ui.com/modules/sidebar.html#using-a-custom-context

Please also see here https://github.com/fomantic/Fomantic-UI/issues/1015

**Comment by ko2in** [comment]
Closed since @lubber-de already explained the solution. Please feel free to reopen if it doesn't resolve.
